webpack/scenes/ContentViews/details/DetailsContainer.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector, shallowEqual } from 'react-redux';
+import PropTypes from 'prop-types';
+import { STATUS } from 'foremanReact/constants';
+import EmptyStateMessage from '../components/EmptyStateMessage';
+import Loading from '../components/Loading';
+import getContentViewDetails from './ContentViewDetailActions';
+import { selectCVDetails,
+  selectCVDetailStatus,
+  selectCVDetailError } from './ContentViewDetailSelectors';
+
+const DetailsContainer = ({ children, cvId, isOpen }) => {
+  const dispatch = useDispatch();
+  const details = useSelector(state => selectCVDetails(state, cvId), shallowEqual);
+  const status = useSelector(state => selectCVDetailStatus(state, cvId), shallowEqual);
+  const error = useSelector(state => selectCVDetailError(state, cvId), shallowEqual);
+
+  useEffect(() => {
+    if (isOpen && Object.keys(details).length === 0) {
+      dispatch(getContentViewDetails(cvId));
+    }
+  });
+
+  if (status === STATUS.PENDING) return (<Loading size="sm" />);
+  // Can we display the error message?
```

Comment:
I don't think this comment is necessary; the next line is pretty self-explanatory :yum:

Reply:
that was actually for me to not forget, with this we don't show the actual API response, I can remove it for now though and file an issue

Reply:
Oh that's fine then, if it's for you to remember something then just stick a TODO in front of it.

Reply:
> just stick a TODO in front of it.

I'm not a fan of these, mostly because they just stick around and nothing happens with them and we have a proper issue tracking system, so I'll just file an issue and associate with the tracker

Reply:
filed! https://projects.theforeman.org/issues/29707